Misc/NEWS.d/next/Library/2023-05-14-11-46-21.gh-issue-87646.DNJjDX.rst

```diff
@@ -0,0 +1,3 @@
+`tempfile.NamedTemporaryFile` and  `tempfile.TemporaryDirectory` includes a
```

Comment:
This should fix the docs build (you'll want to wrap to 80 characters per line, though, and you'll want to do the same for pathlib.Path below)

```suggestion
:class:`tempfile.NamedTemporaryFile` and  :class:`tempfile.TemporaryDirectory` includes a
```

Reply:
Thanks! I have rebased the changes to [ee6435b](https://github.com/python/cpython/pull/104466/commits/ee6435b52832cb6a102391bfddc92883170249f9)